### DISCUSSION TITLE
Add more color steps config

### DISCUSSION THREAD

**Discussion by scottromack** [tailwindlabs/tailwindcss#10616] [Help]
I love tailwind and have used in exclusively on my last 3 projects.
In each one I needed more than the provided steps ( usually something lighter than blue-50 or an in between blue-150 ).

Is this possible already via the config?

**Comment by wongjn** [comment]
Yep! Use `extend` to add additional color keys:
```js
module.exports = {
  theme: {
    colors: {
      blue: {
        150: '<color>',
      },
    },
  },
};
```

**Comment by scottromack** [comment]
I think this just went in in 3.3

**Reply by RobinMalfait** [reply]
We've added a new `950` value by default. But as @wongjn showed you can always add your own values if you want. More info: https://tailwindcss.com/docs/customizing-colors#adding-additional-colors

**Comment by scottromack** [comment]
Well I wanted lighter lights but the above config will work
